repos.json

```diff
@@ -1,275 +1,410 @@
 [
```

Comment:
It would be good to have JSON-Schema for the `repos.json` and `ocrd_all_releases.json`, so we can validate our output and spot issues like the empty release date @paulpestov mentions.

Reply:
Good point! I'll add an issue for that.